keyboards/uranuma/config.h

```diff
@@ -0,0 +1,253 @@
+/*
```

Comment:
Please submit this keyboard in a separate PR.

Reply:
OK. I am preparing